### PR TITLE
VALIDATION image-alt-text-duplicated

### PR DESCRIPTION
**Pull request by mSarts**
No issue number.

### CODE REVIEW

File: aspnet/web-forms/overview/older-versions-security/roles/assigning-roles-to-users-cs.md [hunk 1]
```diff
@@ -176,7 +176,7 @@ Let's augment the `RolesUserList` GridView so that it includes a column of "Remo
 
 Start by adding a Delete button field to the GridView. Make this field appear as the left most filed and change its `DeleteText` property from "Delete" (the default) to "Remove".
 
-[![Add the](assigning-roles-to-users-cs/_static/image20.png)](assigning-roles-to-users-cs/_static/image19.png)
+[![Add a Delete button.](assigning-roles-to-users-cs/_static/image20.png)](assigning-roles-to-users-cs/_static/image19.png)
```
Comment (jmart1428) [inline aspnet/web-forms/overview/older-versions-security/roles/assigning-roles-to-users-cs.md:179]: Let's make it a little more descriptive. Something like "Screenshot that shows how to add the “Remove” button in the Fields window."

File: aspnet/web-forms/overview/older-versions-security/roles/assigning-roles-to-users-vb.md [hunk 2]
```diff
@@ -176,7 +176,7 @@ Let's augment the `RolesUserList` GridView so that it includes a column of "Remo
 
 Start by adding a Delete button field to the GridView. Make this field appear as the left most filed and change its `DeleteText` property from "Delete" (the default) to "Remove".
 
-[![Add the](assigning-roles-to-users-vb/_static/image20.png)](assigning-roles-to-users-vb/_static/image19.png)
+[![Add a Delete button.](assigning-roles-to-users-vb/_static/image20.png)](assigning-roles-to-users-vb/_static/image19.png)
```
Comment (jmart1428) [inline aspnet/web-forms/overview/older-versions-security/roles/assigning-roles-to-users-vb.md:179]: Let's make it a little more descriptive. Something like "Screenshot that shows how to add the “Remove” button in the Fields window."

File: aspnet/web-forms/overview/older-versions-security/roles/assigning-roles-to-users-cs.md [hunk 3]
```diff
@@ -257,7 +257,7 @@ Open the `CreateUserWizardWithRoles.aspx` page and add a CreateUserWizard contro
 
 Next, select the "Add/Remove `WizardSteps`…" option from the CreateUserWizard's Smart Tag and add a new `WizardStep`, setting its `ID` to `SpecifyRolesStep`. Move the `SpecifyRolesStep WizardStep` so that it comes after the "Sign Up for Your New Account" step, but before the "Complete" step. Set the `WizardStep`'s `Title` property to "Specify Roles", its `StepType` property to `Step`, and its `AllowReturn` property to False.
 
-[![Add the](assigning-roles-to-users-cs/_static/image32.png)](assigning-roles-to-users-cs/_static/image31.png)
+[![Add a new Wizard Step.](assigning-roles-to-users-cs/_static/image32.png)](assigning-roles-to-users-cs/_static/image31.png)
```
Comment (jmart1428) [inline aspnet/web-forms/overview/older-versions-security/roles/assigning-roles-to-users-cs.md:260]: Let's make it a little more descriptive. Something like "Screenshot that shows the selected Specify Roles properties in the Wizard Step Collection Editor window."

File: aspnet/web-forms/overview/older-versions-security/roles/assigning-roles-to-users-cs.md (hunk 1 above)
Comment (mSarts) [reply aspnet/web-forms/overview/older-versions-security/roles/assigning-roles-to-users-cs.md:179]: Addressed.

File: aspnet/web-forms/overview/older-versions-security/roles/assigning-roles-to-users-cs.md (hunk 3 above)
Comment (mSarts) [reply aspnet/web-forms/overview/older-versions-security/roles/assigning-roles-to-users-cs.md:260]: Addressed.

File: aspnet/web-forms/overview/older-versions-security/roles/assigning-roles-to-users-vb.md [hunk 4]
```diff
@@ -257,7 +257,7 @@ Open the `CreateUserWizardWithRoles.aspx` page and add a CreateUserWizard contro
 
 Next, select the "Add/Remove `WizardSteps`…" option from the CreateUserWizard's Smart Tag and add a new `WizardStep`, setting its `ID` to `SpecifyRolesStep`. Move the `SpecifyRolesStep WizardStep` so that it comes after the "Sign Up for Your New Account" step, but before the "Complete" step. Set the `WizardStep`'s `Title` property to "Specify Roles", its `StepType` property to `Step`, and its `AllowReturn` property to False.
 
-[![Add the](assigning-roles-to-users-vb/_static/image32.png)](assigning-roles-to-users-vb/_static/image31.png)
+[![Add a new Wizard Step.](assigning-roles-to-users-vb/_static/image32.png)](assigning-roles-to-users-vb/_static/image31.png)
```
[the remark this answers is not in the document]
Comment (mSarts) [reply aspnet/web-forms/overview/older-versions-security/roles/assigning-roles-to-users-vb.md:260]: Addressed.

**Comment by mSarts** [comment]
**Global effort to fix validation errors**

 @Rick-Anderson   

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

**Pull request merged**